src/book/04-guides/03-hardware/02-servo-calibration.mdx

```diff
@@ -0,0 +1,35 @@
+---
+section: Guides
+chapter: Hardware
+title: Servo Calibration
+description: Discover how to calibrate servos.
+slug: /guides/hardware/servo-calibration
+---
+
+Testing
+
+To calibrate the servos please follow the following instructions:
+
+1. Connect the device to your computer and its power supply
+
+2. Connect the servo to the device
+
+3. Open Dynamixel Wizard
+
+4. Tools > Calibrate
+
+5. Read the instructions and click next
+
+6. Choose the correct servo model from the list. **(MX-64 or MX-106 NOT the 2.0 versions)**
+
+7. Select the communications port _(which ones normally?)_
```

Comment:
Yeah, would be useful to have the commonly used port.

Reply:
A "commonly used port" isn't a thing. It will depend on which PC you use and which USB port you plug into (at least on Windows), and which other USB devices you already have plugged in.

On Linux it would probably be ttyUSBX (where X is 0-9, depending on how many other similar devices you might already have plugged in), but it could also be ttyACMX depending on the driver (I think).

No idea what Macs would do, but probably the same Linux.

Reply:
In that case maybe a couple of recommendations on some ports that have worked before  (for Windows/Linux). Or perhaps a link to some resource that someone could use to figure out what to use.

Reply:
Would it be better to recommend that if Dynamixel has more than one option in the dropdown (in my experience it'll just autofill) to just disconnect any unnecessary devices and restart the calibration wizard?

Reply:
If it autofills the right option consistently I reckon just note that and then say if that doesn't work try another one.

Reply:
Another approach for Linux is to inspect the `dmesg` output when you plug the device in to see where it is allocated, though the autofill is probably more convenient